### DISCUSSION TITLE
User manager

### DISCUSSION THREAD

**Discussion by kayvanaarssen** [pglombardo/PasswordPusher#1337] [Ideas]
Manage Active users on an server. When sign-ups are enabled, either with domain validation or Public. 
We would like to be able to manage the users and or delete them / block when needed.
Or even block domains. This of course would need to have Password Pusher with a DB deployed.

**Comment by pglombardo** [comment]
I agree - this is needed.  I created a feature request in #1346.

**Comment by pglombardo** [comment]
Hi @kayvanaarssen - I assume you already saw this but for everyone else, an Admin Dashboard has been added to the project in [v1.39.0](https://github.com/pglombardo/PasswordPusher/releases/tag/v1.39.0) available at `/admin`.

**Reply by kayvanaarssen** [reply]
Yes sir, already made good use of it!
